googletest/test/gtest_catch_exceptions_test_.cc

```diff
@@ -138,7 +138,7 @@ TEST_F(CxxExceptionInConstructorTest, ThrowsExceptionInConstructor) {
 }
 
 // Exceptions in destructors are not supported in C++11.
-#if !defined(__GXX_EXPERIMENTAL_CXX0X__) &&  __cplusplus < 201103L
+#if !GTEST_LANG_CXX11
```

Comment:
So GTEST_SUPPRESS_UNREACHABLE_CODE_WARNING_BELOW_ also needs to disable "it throws even though it says it shouldn't throw" warning?